tests/integration/components/sl-calendar-test.js

```diff
@@ -0,0 +1,773 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+const testContent = Ember.A([
+    {
+        date: new Date( 2022, 8, 17 ),
+        label: 'Today!'
+    }
+]);
+
+const multipleTestContent = Ember.A([
+    {
+        date: new Date( 2022, 8, 17 ),
+        label: 'Event 1 Today!'
+    },
+    {
+        date: new Date( 2022, 8, 17 ),
+        label: 'Event 2 Today!'
+    },
+    {
+        date: new Date( 2022, 8, 17 ),
+        label: 'Event 3 Today!'
+    },
+    {
+        date: new Date( 2022, 8, 20 ),
+        label: 'Event 1 Another Day!'
+    }
+]);
+
+moduleForComponent( 'sl-calendar', 'Integration | Component | sl calendar', {
+    integration: true
+});
+
+test( 'Setting currentYear and currentMonth modifies the view correctly', function( assert ) {
+
+    const currentYear = 2025;
+    this.set( 'currentYear', currentYear );
+
+    const currentMonth = 1;
+    this.set( 'currentMonth', currentMonth );
+
+    this.set( 'content', testContent );
```

Comment:
this.set( 'content', testContent ); is not needed for this test and has been removed